src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/GenericDefinitionEETypeNode.cs

```diff
@@ -24,6 +24,16 @@ public override bool ShouldSkipEmittingObjectNode(NodeFactory factory)
 
         protected override DependencyList ComputeNonRelocationBasedDependencies(NodeFactory factory)
         {
+            // Generated type contains generic virtual methods that will get added to the GVM tables.
+            // The entries in the GVM tables are expressed in open generic definitions, so any generic 
+            // definition type that has generic virtual methods will need some dependency tracking.
+            if (TypeGVMEntriesNode.TypeNeedsGVMTableEntries(_type))
```

Comment:
I see. I'll remove this from here. This will definitely need testing when multimodule mode is enabled

---
In reply to: [96538983](https://github.com/dotnet/corert/pull/2521#discussion_r96538983) [](ancestors = 96538983)